localisation/00_wc_artifact_localisation.csv

```diff
@@ -32,61 +39,122 @@ libram_of_protection_key;This libram is among the church's most ancient relics.
+great_royal_sword;Great Royal Sword;;Gro�es k�nigliches Schwert;;;;;;;;;;;
+great_royal_sword_desc;Great Royal Sword;;Gro�es k�nigliches Schwert;;;;;;;;;;;
+legion_doomhorn;Legion Doomhorn;;Schreckenshorn der Legion;;;;;;;;;;;x
+legion_doomhorn_desc;The inside of a doom horn looks like it has been blackened with a flame, but feels smooth and cool. The sound of this horn freezes the blood in the veins of the enemies.;;Das Innere eines Schreckenshorns wirkt wie mit einer Flamme geschw�rzt, f�hlt sich aber glatt und k�hl an. Der Ton dieses Horns l�sst das Blut in den Adern der Feinde gefrieren.;;;;;;;;;;;x
 libram_of_devoutness_03;Libram of Devoutness;;Foliant der Fr�mmigkeit;;;;;;;;;;;x
+libram_of_devoutness_03_desc;This libram is a symbol of devoutness to the ideals of the holy light. Its owner proves to be a faithful student of the light and a righteous defender.;;Dieser Foliant ist ein Symbol der Fr�mmigkeit vor den Idealen des heiligen Lichts. Sein Besitzer erweist sich als treuer Sch�ler und rechtschaffener Verteidiger des Lichts.;;;;;;;;;;;x
 libram_of_guardianship_03;Libram of Guardianship;;Foliant der Bewahrung;;;;;;;;;;;x
+libram_of_guardianship_03_desc;This libram is a symbol of guardianship before the ideals of the holy light. Its owner proves to be a strong and righteous fighter for the light.;;Dieser Foliant ist ein Symbol der Bewahrung vor den Idealen des heiligen Lichts. Sein Besitzer erweist sich als starker und gerechter K�mpfer f�r das Licht.;;;;;;;;;;;x
 libram_of_reckoning_03;Libram of Reckoning;;Foliant der Abrechnung;;;;;;;;;;;x
+libram_of_reckoning_03_desc;This libram is a symbol of reckoning before the ideals of the holy light. Its owner proves to be a strong fighter for the light.;;Dieser Foliant ist ein Symbol der Abrechnung vor den Idealen des heiligen Lichts. Sein Besitzer erweist sich als starker K�mpfer f�r das Licht.;;;;;;;;;;;x
 libram_of_righteousness_03;Libram of Righteousness;;Foliant der Rechtschaffenheit;;;;;;;;;;;x
+libram_of_righteousness_03_desc;This libram is a symbol of righteousness before the ideals of the holy light. Its owner proves to be a faithful student and righteous fighter for the light.;;Dieser Foliant ist ein Symbol der Rechtschaffenheit vor den Idealen des heiligen Lichts. Sein Besitzer erweist sich als treuer Sch�ler und rechtschaffener K�mpfer f�r das Licht.;;;;;;;;;;;x
+lionhorn_of_stormwind;Lionhorn of Stormwind;;L�wenhorn von Stormwind;;;;;;;;;;;x
+lionhorn_of_stormwind_desc;The powerful sound of this horn makes the hearts of the soldiers beat higher.;;Der kraftvolle Klang dieses Horns l�sst die Herzen der Soldaten h�her schlagen.;;;;;;;;;;;x
```

Comment:
I don't think you can make to beat a heart "higher". Generally in this context I'd go for either:

- harder
- faster
- louder (if you want it to be 'inspiring', nobody in battle is going to hear this logically)

Reply:
done